### ISSUE TITLE
shairport stdout 8 bit rate 8000 Hz Mono

### ISSUE THREAD

**Issue by CarlBernard** [mikebrady/shairport-sync#1301]
![F2256622-667F-415C-8937-776CE699E389](https://user-images.githubusercontent.com/85563569/135158897-e18f5b2c-024e-419e-a16d-282f06621887.jpeg)

Is it normal that if I send via airplay from iPad in lossless format to shairport-sync + stdout I have this output with aplay when I play the raw file (see screenshot)?

I have compared the file on Audition to the original one and there are obvious differences

**Comment by mikebrady** [comment]
Thanks for the post. There are a couple of things here.

Firstly, the output from the stdout is not a WAV file. It’s simply the raw frames sequence. 

Secondly, it has been established that iTunes does transmit losslessly to Shairport Sync. However, we don’t know what the iPad may do before transmitting the audio.

**Comment by CarlBernard** [comment]
646 / 5000
Risultati della traduzione
Hello Mike,

the problem is not with stdout which regularly creates a raw file via shairport-sync.
The "real problem" is that tvOS on Apple TV 4K, unlike iOS, is not bit perfect after the tests I did comparing the tracks with Audition.

I have compared two tracks and unfortunately they do not cancel each other out. The sound is partially reduced, however both in listening and in frequency analysis it is evident that the tracks are not the same.
 
Mine was just a doubt.

Anyway thanks for the quick reply and thanks again for all your work on shairport-sync.

P.S: news for Airplay 2? :-)) Is there already a version in development?

Have a good evening
